Test/git-issues/git-issue-1665.dfy.expect

```diff
@@ -165,13 +165,44 @@ module ConcreteModule {
       }      
     } else {
       return Success;
-    }    ---------- end desugared ----------*/
+    }    ---------- end flattened ----------*/
     match n {
       case {:split false} 0 =>
-        :- Func1();
+        :- Func1()
```

Comment:
The `;` is part of the statement, but it's only shown a few lines further ahead because the printing of the expression is multi line since it includes the 'flattened' part.